canonical-web-and-design/ubuntu.com
# Convert templates to Jinja2

As the first step in the Flask conversion, this is to convert the templates to Jinja2 syntax.

Fixes https://github.com/canonical-web-and-design/base-squad/issues/682

## QA

- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Click around the site *a lot*, check nothing's wrong
